### ISSUE TITLE
AdGuard Server Proxy

### ISSUE THREAD

**Issue by ynnniV** [AdguardTeam/AdguardFilters#163501]
### Prerequisites

- [X] This site DOES NOT contain sexually explicit material, otherwise use [NSFW-specific form](./new?template=bug_report_NSFW.yml);
- [X] Filters were updated before reproducing an issue;
- [X] AdGuard product version is up-to-date;
- [X] Browser version is up-to-date;
- [X] If the site or app is broken, disabling AdGuard protection resolves an issue.

### What product do you use?

AdGuard for Windows

### AdGuard version

7.15

### What type of problem have you encountered?

Website or app doesn't work properly

### Which browser(s) do you use?

the problem is with app, not a site

### Which device do you use?

Desktop

### Where is the problem encountered?

There is no site, but Microsoft Store and Whatsapp

### What filters do you have enabled?

AdGuard Base filter, AdGuard Russian filter, AdGuard Spanish/Portuguese filter, EasyList Italy

### Add your comment and screenshots

1. I've added the adguard server proxy to the Windows one for filter all things on my PC. I'll give you two screenshot. One of Microsoft Store where is loading nothing and the second one is Whatsapp where i can message but i cannot download the images. I tried to watch the logs on AdGuard, but there is nothing showing when i try to open Microsoft Store or Whatsapp downloading images.

2. Screenshots
<details><summary>Screenshot 1:</summary>
Screen Microsoft Store: 

![image](https://github.com/AdguardTeam/AdguardFilters/assets/9847632/7be940cf-e01e-4f6b-a759-d24d40fb9a6d)
</details>

<details><summary>Screenshot 2:</summary>
Screen Whatsapp: 

![image](https://github.com/AdguardTeam/AdguardFilters/assets/9847632/628a7e0f-d361-436c-9bb5-8e488d2028a6)

</details>

### Privacy

- [X] I agree to follow this condition

**Comment by ynnniV** [comment]
If the screenshot is not showing, there is the link of gyazo:

- Screen Microsoft Store: https://i.gyazo.com/2552cc0760bf898b37a7802f2c65fcc2.png
- Screen Whatsapp: https://i.gyazo.com/ab6d4213e2a5ec7def54e1c9194a2513.png

**Comment by ynnniV** [comment]
If the screenshot is not showing, there is the link of gyazo:

- Screen Microsoft Store: https://i.gyazo.com/2552cc0760bf898b37a7802f2c65fcc2.png
- Screen Whatsapp: https://i.gyazo.com/ab6d4213e2a5ec7def54e1c9194a2513.png

**Comment by ynnniV** [comment]
Solved whatsapp enabling on settings -> advanced settings -> "use transfer mode" enabled. But Microsoft Store still having problem to open up.

**Comment by zloyden** [comment]
Hi! Could you check it with disabled filters, leaving the protection on?

**Comment by ynnniV** [comment]
> Hi! Could you check it with disabled filters, leaving the protection on?

Hey! Thanks for reply!
I tried again, for making all sure of that problem.

- Microsoft Store, still not loading with filter disabled.
- Whatsapp (App of Windows): still not loading audio or images.

With Web Whatsapp on browser there is no problem.

If i remove the protection all is not working cause of the proxy. 

Tried to remove the DNS filter too but nothing change.
